app/models/authentication_request_header.rb

```diff
@@ -0,0 +1,11 @@
+class AuthenticationRequestHeader < Authentication
+  def to_ansible_config_format
```

Comment:
This name still feels wrong...For one, the name implies that it's a format that makes sense for *any* kind of ansible deployment, but I think this is very specific to container deployment.  As such, it feels like it doesn't even belong here at all...these models are generic and should not have specific usages inside of them.  I think it would make a lot more sense living inside the container deployment model...you already hardcode a list of Authentication models, and this seems like an extension to most of that.